python/ray/autoscaler/node_provider.py

```diff
@@ -66,23 +61,14 @@ def load_azure_example_config():
         os.path.dirname(ray_azure.__file__), "example-full.yaml")
 
 
-def import_external():
-    """Mock a normal provider importer."""
-
-    def return_it_back(config):
-        return config
-
-    return return_it_back, None
-
-
 NODE_PROVIDERS = {
     "local": import_local,
     "aws": import_aws,
     "gcp": import_gcp,
     "azure": import_azure,
     "kubernetes": import_kubernetes,
     "docker": None,
-    "external": import_external  # Import an external module
+    "external": None  # Import an external module
```

Comment:
Can we also make this the same by moving the load class stuff from get_node_provider() into a import function like the rest?

Reply:
(passing in the config to the import_foo functions as import_foo(config) would enable this).

Reply:
I added that, @ericl. I just don't like that we have to pass provider_config to the other importers.
You can merge the PR.